pom.xml

```diff
@@ -16,12 +16,12 @@
   <parent>
     <groupId>org.pentaho</groupId>
     <artifactId>pentaho-ce-jar-parent-pom</artifactId>
-    <version>9.2.0.1-364</version>
+    <version>9.6.0.0-SNAPSHOT</version>
```

Comment:
This won't be releasable with references to SNAPSHOT versions for a parent.  Can you pick the 9.5.0.0 GA instead?